doc/classes/Color.xml

```diff
@@ -510,7 +522,7 @@
 			Bisque color.
 		</constant>
 		<constant name="BLACK" value="Color(0, 0, 0, 1)">
-			Black color.
+			Black color. The default value of any color.
```

Comment:
Note that this is not true in C# since a default initialized struct has all their fields initialized to zero, which means:

```csharp
Color color;
// All these result in the same color value:
color = default;
color = new Color();
color = new Color(0, 0, 0, 0);
```

I don't know if this needs to be mentioned here though. I think we already mention in the `C# differences` documentation page that structs are by default zero-initialized so it may not correspond to the default value in GDScript.

Reply:
Funny. I opened an issue about this: https://github.com/godotengine/godot/issues/69009...

Perhaps it's worth bringing the C# inconsistency up, too.